src/components/TextField/components/Resizer/tests/Resizer.test.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+import {mountWithAppProvider, findByTestID, trigger} from 'test-utilities';
+import {noop} from '@shopify/javascript-utilities/other';
+import Resizer from '../Resizer';
+import EventListener from '../../../../EventListener';
+
+describe('<Resizer />', () => {
+  const mockProps = {
+    onHeightChange: noop,
+    contents: 'Contents',
+  };
+
+  describe('contents', () => {
+    it('renders contents', () => {
+      const contents = 'Contents';
+      const resizer = mountWithAppProvider(
+        <Resizer {...mockProps} contents={contents} />,
+      );
+      const contentsNode = findByTestID(resizer, 'ContentsNode');
+      const expectedHTML = '<div testid="ContentsNode">Contents<br></div>';
+      expect(contentsNode.html()).toBe(expectedHTML);
+    });
+
+    it('properly encodes HTML entities', () => {
+      const contents = `<div>&\nContents</div>`;
+      const resizer = mountWithAppProvider(
+        <Resizer {...mockProps} contents={contents} />,
+      );
+      const contentsNode = findByTestID(resizer, 'ContentsNode');
+      const expectedEncodedContents =
+        '&lt;div&gt;&amp;<br>Contents&lt;/div&gt;<br></div>';
+      expect(contentsNode.html()).toContain(expectedEncodedContents);
+    });
+  });
+
+  describe('minimumLines', () => {
+    it('renders a number of <br> tags equivalent to minimumLines', () => {
+      const minimumLines = 3;
+      const resizer = mountWithAppProvider(
+        <Resizer {...mockProps} minimumLines={minimumLines} />,
+      );
+      const breakingSpaces = findByTestID(resizer, 'MinimumLines');
+      expect(breakingSpaces.html()).toContain('<br><br><br>');
+    });
+  });
+
+  describe('onHeightChange()', () => {
+    it('is called on componentDidMount() if minimumLines is provided', () => {
+      const spy = jest.fn();
+      const minimumLines = 3;
+      mountWithAppProvider(
+        <Resizer
+          {...mockProps}
+          currentHeight={50}
```

Comment:
Why does `currentHeight` need to be defined for this?

Reply:
Because it needs a height to compare with. https://github.com/Shopify/polaris-react/blob/master/src/components/TextField/components/Resizer/Resizer.tsx#L71